NEWS.md

```diff
@@ -1,6 +1,6 @@
 # leaflet (development version)
 
-* Color palette improvements. All color palette functions now support all `{viridisLite}` palettes ("magma", "inferno", "plasma", "viridis", "cividis", "rocket", "mako", and "turbo").
+* Color palette improvements. All color palette functions now support all `{viridisLite}` palettes ("magma", "inferno", "plasma", "viridis", "cividis", "rocket", "mako", and "turbo") (@jack-davison, #924).
```

Comment:
```suggestion
* Color palette improvements. All color palette functions now support all `{viridisLite}` palettes ("magma", "inferno", "plasma", "viridis", "cividis", "rocket", "mako", and "turbo") (@jack-davison, #924).

* Update vignettes to replace `{sp}` usage with `{sf}` and fix their broken examples. (@jack-davison, #928)
```

Reply:
`{terra} - missing closing back-tick! Should be `{terra}`